src/it/resources/expected/my_enum/python/my_enum.py

```diff
@@ -6,9 +6,8 @@
 
 from djinni import exception # this forces run of __init__.py which gives cpp option to call back into py to create exception
 
-from enum import IntEnum, unique
+from enum import IntEnum
 
-@unique
```

Comment:
Is there a reason why the `@unique` annotation had to be removed? Is it incompatible with `IntFlag`?

Reply:
I concluded [from the docs](https://docs.python.org/3/library/enum.html) it only applies to `Enum` and since the code is generated anyways I didn't see the value of having it (even if it was applicable).